pkg/cmd/init.go

```diff
@@ -214,7 +214,8 @@ func certSetup(cfg *config.MicroshiftConfig) (*certchains.CertificateChains, err
 					ValidityDays: cryptomaterial.ShortLivedCertificateValidityDays,
 				},
 				Hostnames: []string{
-					"router-default.apps." + cfg.Cluster.Domain,
+					"router-default.apps." + cfg.BaseDomain,
+					"*.apps." + cfg.BaseDomain, // wildcard for any additional auto-generated domains
```

Comment:
I checked on an full OpenShift:

[majopela@rhel9 microshift]$ openssl s_client -connect oauth-openshift.apps.ci-ln-n3z62kb-76ef8.aws-2.ci.openshift.org:443  </dev/null 2>/dev/null | openssl x509 -noout -text | grep DNS
                DNS:*.apps.ci-ln-n3z62kb-76ef8.aws-2.ci.openshift.org

Probably we can remove 217 as it is redundant,

Reply:
@stlaz and @oglok are our cert experts, let's get them to confirm.

Reply:
no, SNO / OpenShift uses a wildcard cert *.apps.clustername.basedomain

that means that any other applications exposed under that domain will have a "valid" (self signed, but OK), certificate...

Reply:
See my check above @fzdarsky :

```
[majopela@rhel9 microshift]$ openssl s_client -connect oauth-openshift.apps.ci-ln-n3z62kb-76ef8.aws-2.ci.openshift.org:443 </dev/null 2>/dev/null | openssl x509 -noout -text | grep DNS
DNS:*.apps.ci-ln-n3z62kb-76ef8.aws-2.ci.openshift.org
```

Reply:
Sorry for not being clear: I was trying to point out the missing `cfg.ClusterName` part (i.e. `ci-ln-n3z62kb-76ef8`). You're right, of course, that this is not only for the `router-default.aps.` line but also for the `*.apps.` line.